lib/moesif_rack/app_config.rb

```diff
@@ -45,13 +45,13 @@ def parse_configuration(config_api_response)
             else
                 @helpers.log_debug 'Response body is nil, assuming default behavior'
                 # Response body is nil, so assuming default behavior
-                return nil, 100, Time.now.utc
+                return nil, nil, Time.now.utc
```

Comment:
@dgilling please double check this is expected signature/response. I verified after looking/debugging parseConfiguration method.